### PR TITLE
Adds a powerhead and the crafting recipe

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Content "Adds a new weapon - the powerhead"

#### Purpose of change

I wanted to make something like the speargun but with the exception that it is equally effective on land. It also uses a non-firearm skill.
#### Describe the solution

It's a powerhead. This one is a makeshift version consisting of a pipe gun taped to a long stick/plank. Useful for pesky sewer gators or zombie sharks.

#### Describe alternatives you've considered

None.

#### Testing

I suggest shooting a z-shark or two after spawning the powerhead. See if it works. Might need a range of 2 to make it usable like a spear.

### CODE REVIEW

File: data/json/items/ranged/spearguns.json [hunk 1]
```diff
@@ -203,5 +203,28 @@
...
+    "price_postapoc": 1000,
+    "bashing": 4,
+    "skill": "stabbing",
+    "material": [ "steel", "wood" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": -8 },
+    "dispersion": 660,
+    "durability": 4,
+    "clip_size": 1,
+    "reload": 600,
+    "modes": [ [ "DEFAULT", "single", 1 ],
```
Comment (LaVeyanFiend) [inline data/json/items/ranged/spearguns.json:226]:
```suggestion
    "modes": [ [ "DEFAULT", "single", 1 ] ],
```

File: data/json/items/ranged/spearguns.json [hunk 2]
```diff
@@ -203,5 +203,28 @@
...
+    "flags": [ "RELOAD_EJECT", "SHEATH_SPEAR", "FRAGILE_MELEE" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "shot": 1 } } ]
+  },
```
Comment (LaVeyanFiend) [inline data/json/items/ranged/spearguns.json:229]:
```suggestion
  }
```

File: data/json/recipes/weapon/ranged.json [hunk 3]
```diff
@@ -489,5 +489,20 @@
...
+    "skill_used": "mechanics",
+    "difficulty": "1",
+    "skills_required": [ [ "gun", 1 ] ],
+    "time": "30 min",
+    "autolearn": true,
+    "qualities": [
+      { "id": "SAW_M", "level": 2 },
+    ],
+    "components": [ [ [ "slamfire_shotgun", 1 ] ], [ [ "wire", 1 ] ], [ [ "plank", 1 ] ],[ [ "duct_tape", 100 ] ] ]
+  },
```
Comment (LaVeyanFiend) [inline data/json/recipes/weapon/ranged.json:507]:
```suggestion
  }
```

File: data/json/items/ranged/spearguns.json [hunk 4]
```diff
@@ -203,5 +203,30 @@
...
+    "id": "powerhead",
+    "type": "GUN",
+    "name": { "str": "powerhead" },
+    "description": "A single-shot firearm mounted on a stick.  Commonly used for spear fishing or for defense against sharks.  This one has an activation pin the must be pulled to fire.",
+    "weight": "4000 g",
+    "volume": "4765 ml",
+    "longest_side": "165 cm",
+    "looks_like": "pipe",
+    "price": 8500,
+    "range": 2,
```
Comment (LaVeyanFiend) [inline data/json/items/ranged/spearguns.json:217]: This needs to be negative to lower the range of shooting, it's a modifier to the range granted by the ammo you load it with(00 shot is 12, dragon's breath is 6, slugs are 24).

Comment (Xenomorph-III) [reply data/json/items/ranged/spearguns.json:217]: Ah, OK. I intended the shooting range to be 2 tiles like a spear.
So I'll try -10 as a start.

File: data/json/recipes/weapon/ranged.json [hunk 5]
```diff
@@ -489,5 +489,18 @@
...
+  {
+    "type": "recipe",
+    "result": "powerhead",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "mechanics",
+    "difficulty": "1",
+    "skills_required": [ [ "gun", 1 ] ],
+    "time": "30 min",
```
Comment (kevingranade) [inline data/json/recipes/weapon/ranged.json:501]: Load error here

File: data/json/recipes/weapon/ranged.json [hunk 6]
```diff
@@ -489,5 +489,18 @@
...
     ],
     "components": [ [ [ "pipe", 4 ] ], [ [ "nail", 2 ] ], [ [ "pipe_fittings", 2 ] ], [ [ "scrap", 3 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "powerhead",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "mechanics",
+    "difficulty": "1",
```
Comment (Fris0uman) [inline data/json/recipes/weapon/ranged.json:499]:
```suggestion
    "difficulty": 1,
```

File: data/json/items/ranged/spearguns.json [hunk 7]
```diff
@@ -203,5 +203,32 @@
...
     ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "fishspear": 1 } } ]
+  },
+  {
+    "id": "powerhead",
+    "type": "GUN",
+    "name": { "str": "powerhead" },
+    "symbol": "(",
+    "color": "brown",
+    "description": "A single-shot firearm mounted on a stick.  Commonly used for spear fishing or for defense against sharks.  This one has an activation pin the must be pulled to fire.",
```
Comment (LaVeyanFiend) [inline data/json/items/ranged/spearguns.json:213]:
```suggestion
    "description": "A single-shot firearm mounted on a stick.  Commonly used for spear fishing or for defense against sharks.  This one has an activation pin that must be pulled to fire.",
```

**Comment by Xenomorph-III** [comment]
Closing this till KorGgenT does his stuff.

**Pull request closed without merging**